testing/tools/external_sender/external_sender.pony

```diff
@@ -53,7 +53,8 @@ actor Main
               -----------------------------------------------------------------------------------
               --external/-e [Specifies address to send message to]
               --type/-t [Specifies message type]
-                  clean-shutdown | rotate-log | partition-query | print
+                  clean-shutdown | rotate-log | partition-query |
+                  cluster-status-query | print
```

Comment:
I'm wondering if this would be better as just `status`, and maybe adding a separate `worker-status` at a later time for individual worker status.